src/main/java/com/ericsson/ei/controller/model/Subscription.java

```diff
@@ -153,6 +159,26 @@ public void setUserName(String userName) {
         this.userName = userName;
     }
 
+    @JsonProperty("restUser")
```

Comment:
We have Lombok lib for getter and setter, just use annotation @Getter and @Setter, something like
@JsonProperty("token")
@Getter @Setter
private String token;